src/injected/analyzers/focus-traps-handler.ts

```diff
@@ -26,11 +26,15 @@ export class FocusTrapsHandler {
         }
 
         const elementFocusedBeforeTab = this.lastFocusedElement;
-        this.lastFocusedElement = dom.activeElement;
+        this.lastFocusedElement = this.getCurrentFocusedElement(dom.activeElement);
 
         await this.promiseFactory.delay(null, this.keyboardTrapTimeout);
 
-        const currentFocusedElement = dom.activeElement;
+        const currentFocusedElement = this.getCurrentFocusedElement(dom.activeElement);
+
+        if (currentFocusedElement === dom.body) {
+            return null;
```

Comment:
Line 36 is the only one without test coverage in injected/analyzers; consider adding a unit test?